addons/web/static/src/views/kanban/progress_bar_hook.js

```diff
@@ -152,6 +155,16 @@ class ProgressBarState {
                     0;
             }
         }
+
+        if (aggregateField.currency_field && group.list.records.length > 0) {
+            const record = group.list.records[0];
+            const currencyId = record.data[aggregateField.currency_field]
+            currency = currencyId ? getCurrency(currencyId[0]) : false;
+            if (currency) {
+                session.company_currency_id = currencyId[0];
+                return { title, value, currency };
+            }
+        }
```

Comment:
<img width="545" height="624" alt="Screenshot 2025-10-15 at 5 34 31 PM" src="https://github.com/user-attachments/assets/acfb2774-091e-4cb9-9174-7ff5fbaa197b" />
Bên crm cũng đang dùng đến session.company_currency_id nên em update ở đây để cả cột số liệu thứ 2 mà bên crm thêm cũng hiển thị đúng format.

- Nếu muốn an toàn hơn thì có thể sửa AnimatedNumber cho phép truyền thêm prop tiền tệ